test/unit/object/test_web_link.py

```diff
@@ -0,0 +1,50 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+
+
+def test_get(mock_box_session, test_web_link):
+    # pylint:disable=redefined-outer-name, protected-access
+    web_link_id = test_web_link.object_id
+    expected_url = mock_box_session.get_url('web_links', web_link_id)
```

Comment:
We should not use the convenience URL methods in tests, otherwise there's nothing verifying that the implementation of `get_url()` is correct.  Please construct the expected URL literally to verify